client/src/components/Switch/index.js

```diff
@@ -0,0 +1,3 @@
+import Switch from './Switch';
+
```

Comment:
Please remove this file.
No need the index.js, you can import Switch directly from components/Switch/Switch.js
(in addition change the jsx to js)